src/components/protonMedal.tsx

```diff
@@ -73,7 +72,7 @@ export default function ProtonMedal(
jMtNy4zLDEwNC43LDE4LjUsMTE5LjZjNiwzLjUsMTIuOSw1LjEsMjAuNiw1LjEKCQljMjUuMiwwLDU4LjItMTgsOTEuNi00OS4yYzMzLjQsMzEsNjYuNCw0OC44LDkxLjYsNDguOGM3LjcsMCwxNC43LTEuNywyMC43LTUuMWMyNS44LTE0LjksMzEuNi02MS4yLDE4LjMtMTE5LjMKCQlDNDMzLjQsMzAzLjUsNDcwLjUsMjc1LjMsNDcwLjUsMjQ1LjV6IE0zNTEuMSwxODQuNGMtMy40LDExLjgtNy42LDI0LTEyLjQsMzYuMmMtMy44LTcuMy03LjctMTQuNy0xMi0yMgoJCWMtNC4yLTcuMy04LjctMTQuNS0xMy4yLTIxLjVDMzI2LjUsMTc5LDMzOS4xLDE4MS40LDM1MS4xLDE4NC40eiBNMzA5LjEsMjgyLjFjLTcuMiwxMi40LTE0LjUsMjQuMS0yMi4xLDM1CgkJYy0xMy43LDEuMi0yNy41LDEuOC00MS41LDEuOGMtMTMuOSwwLTI3LjctMC42LTQxLjMtMS43Yy03LjYtMTAuOS0xNS0yMi42LTIyLjItMzQuOWMtNy0xMi0xMy4zLTI0LjItMTkuMS0zNi41CgkJYzUuNy0xMi4zLDEyLjEtMjQuNiwxOS0zNi42YzcuMi0xMi40LDE0LjUtMjQuMSwyMi4xLTM1YzEzLjctMS4yLDI3LjUtMS44LDQxLjUtMS44YzEzLjksMCwyNy43LDAuNiw0MS4zLDEuNwoJCWM3LjYsMTAuOSwxNSwyMi42LDIyLjIsMzQuOWM3LDEyLDEzLjMsMjQuMiwxOS4xLDM2LjVDMzIyLjMsMjU3LjcsMzE1LjksMjcwLDMwOS4xLDI4Mi4xeiBNMzM4LjcsMjcwLjFjNSwxMi4zLDkuMiwyNC42LDEyLjcsMzYuNQoJCWMtMTIsMi45LTI0LjcsNS40LTM3LjgsNy4zYzQuNS03LjEsOS0xNC4zLDEzLjItMjEuN0MzMzEsMjg0LjksMzM0LjksMjc3LjUsMzM4LjcsMjcwLjF6IE0yNDUuNywzNjhjLTguNS04LjgtMTcuMS0xOC42LTI1LjUtMjkuNAoJCWM4LjMsMC40LDE2LjcsMC42LDI1LjIsMC42YzguNiwwLDE3LjItMC4yLDI1LjUtMC42QzI2Mi43LDM0OS40LDI1NC4xLDM1OS4yLDI0NS43LDM2OHogTTE3Ny40LDMxNGMtMTMtMS45LTI1LjYtNC4zLTM3LjYtNy4yCgkJYzMuNC0xMS44LDcuNi0yNCwxMi40LTM2LjJjMy44LDcuMyw3LjcsMTQuNywxMiwyMkMxNjguNSwyOTkuOCwxNzIuOSwzMDcsMTc3LjQsMzE0eiBNMjQ1LjIsMTIzLjFjOC41LDguOCwxNy4xLDE4LjYsMjUuNSwyOS40CgkJYy04LjMtMC40LTE2LjctMC42LTI1LjItMC42Yy04LjYsMC0xNy4yLDAuMi0yNS41LDAuNkMyMjguMywxNDEuNywyMzYuOCwxMzEuOSwyNDUuMiwxMjMuMXogTTE3Ny4zLDE3Ny4xCgkJYy00LjUsNy4xLTksMTQuMy0xMy4yLDIxLjdjLTQuMiw3LjMtOC4yLDE0LjctMTEuOSwyMmMtNS0xMi4zLTkuMi0yNC42LTEyLjctMzYuNUMxNTEuNiwxODEuNSwxNjQuMiwxNzksMTc3LjMsMTc3LjF6IE05NC4zLDI5MgoJCWMtMzIuNS0xMy45LTUzLjUtMzItNTMuNS00Ni40YzAtMTQuNCwyMS0zMi43LDUzLjUtNDYuNGM3LjktMy40LDE2LjUtNi40LDI1LjQtOS4zYzUuMiwxOCwxMi4xLDM2LjcsMjAuNiw1NS45CgkJYy04LjQsMTkuMS0xNS4yLDM3LjctMjAuNCw1NS42QzExMC45LDI5OC41LDEwMi4zLDI5NS40LDk0LjMsMjkyeiBNMTQzLjcsNDIzYy0xMi41LTcuMi0xNy45LTM0LjQtMTMuNy02OS40CgkJYzEtOC42LDIuNy0xNy43LDQuNy0yN2MxOCw0LjQsMzcuNiw3LjgsNTguMiwxMGMxMi40LDE3LDI1LjIsMzIuNCwzOC4yLDQ1LjljLTI5LjksMjcuOC01OCw0My03Nyw0MwoJCUMxNDkuOSw0MjUuNCwxNDYuNCw0MjQuNiwxNDMuNyw0MjN6IE0zNjEuMywzNTMuMWM0LjMsMzUtMSw2Mi4zLTEzLjQsNjkuNWMtMi44LDEuNy02LjMsMi40LTEwLjUsMi40Yy0xOSwwLTQ3LjEtMTUuMS03Ny00Mi43CgkJYzEyLjgtMTMuNSwyNS43LTI4LjgsMzcuOS00NS44YzIwLjctMi4yLDQwLjQtNS42LDU4LjMtMTAuMUMzNTguNiwzMzUuNywzNjAuMiwzNDQuNiwzNjEuMywzNTMuMXogTTM5Ni42LDI5MgoJCWMtNy45LDMuNC0xNi41LDYuNC0yNS40LDkuM2MtNS4yLTE4LTEyLjEtMzYuNy0yMC42LTU1LjljOC40LTE5LjEsMTUuMi0zNy43LDIwLjQtNTUuNmM5LjEsMi44LDE3LjcsNiwyNS44LDkuNAoJCWMzMi41LDEzLjksNTMuNSwzMiw1My41LDQ2LjRDNDUwLDI1OS45LDQyOSwyNzguMiwzOTYuNiwyOTJ6Ii8+Cgk8cGF0aCBjbGFzcz0ic3QxIiBkPSJNMTUzLjYsNDUuNUwxNTMuNiw0NS41TDE1My42LDQ1LjV6Ii8+Cgk8Y2lyY2xlIGNsYXNzPSJzdDIiIGN4PSIyNDUuNCIgY3k9IjI0NS41IiByPSI0MS45Ii8+Cgk8cGF0aCBjbGFzcz0ic3QxIiBkPSJNMzM2LjgsNDUuMkwzMzYuOCw0NS4yTDMzNi44LDQ1LjJ6Ii8+CjwvZz4KPC9zdmc+Cg=='" />
```

Comment:
Does the single quote at the end of this line serve a purpose?

`Cg=='"`

it seems to break the image for me

Reply:
not sure how that snuck in there, removing it seems to fix it?